### PR TITLE
Vue: add Custom Bundle topic

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: concepts/55 Vue Components/05 Add DevExtreme to a Vue Application/64 Create a Custom Bundle.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    Your bundle name.
+
+    - `outDir: 'devextreme-bundle'`    
+    A folder for the generated bundle. The default value is `dist`.
+
+    Leave `vite.config.js` file as it is to build Vue applications separately.
+
+3. Configure your `package.json`.
+
+        <!-- tab: package.json -->
```
Comment (ivanblinov2k17) [inline concepts/55 Vue Components/05 Add DevExtreme to a Vue Application/64 Create a Custom Bundle.md:67]: add type: "module" to package.json configuration

**Review by ivanblinov2k17 — APPROVED** [review]

File: concepts/55 Vue Components/05 Add DevExtreme to a Vue Application/64 Create a Custom Bundle.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+In certain scenarios, you may want to utilize a custom bundle rather than an automatically generated one. The example below illustrates how to implement a custom bundle with Vite.
+
+#include btn-open-github with {
+    href: "https://github.com/DevExpress-Examples/devextreme-vite-vue-bundling"
+}
+
+For best experience, we recommend you to use the latest version of DevExtreme alongside ESM.
```
Comment (dirk-pieterse) [inline concepts/55 Vue Components/05 Add DevExtreme to a Vue Application/64 Create a Custom Bundle.md]:
```suggestion
For the best experience, we recommend that you use the latest version of DevExtreme alongside ESM.
```

File: concepts/55 Vue Components/05 Add DevExtreme to a Vue Application/64 Create a Custom Bundle.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                        globals: {
+                            vue: 'Vue',
+                        },
+                    },
+                },
+                outDir: 'devextreme-bundle'
+            },
+        })
+
+    In the code snippet above, the following options are presented:
```
Comment (dirk-pieterse) [inline concepts/55 Vue Components/05 Add DevExtreme to a Vue Application/64 Create a Custom Bundle.md]:
```suggestion
    In the code snippet above, the following options are used:
```

File: concepts/55 Vue Components/05 Add DevExtreme to a Vue Application/64 Create a Custom Bundle.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+    - `esmExternals: true`     
+    Treat external dependencies (Vue in this case) as ECMAScript modules.
+
+    - `entry: resolve(__dirname, 'main.js')`    
+    A main entry file. We created it at the step one.
```
Comment (dirk-pieterse) [inline concepts/55 Vue Components/05 Add DevExtreme to a Vue Application/64 Create a Custom Bundle.md]:
```suggestion
    A main entry file created at step one.
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**